### PR TITLE
feat: Add mobile command for emulator console feature

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Related to https://github.com/appium/appium/issues/6028

Based on https://github.com/appium/appium-adb/pull/517

### CODE REVIEW

File: lib/commands/emu-console.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * Executes a command through emulator telnet console interface and returns its output.
+ * The `emulator_console` server feature must be enabled in order to use this method.
+ *
+ * @param {ExecOptions} opts
+ * @returns {string} The command output
+ * @throws {Error} If there was an error while connecting to the Telnet console
+ * or if the given command returned non-OK response
+ */
+commands.mobileExecEmuConsoleCommand = async function mobileExecEmuConsoleCommand (opts = {}) {
+  this.ensureFeatureEnabled(EMU_CONSOLE_FEATURE);
```
Comment (KazuCocoa) [inline lib/commands/emu-console.js:30]: 👍

**Comment by mivola** [comment]
Which version/release will contain this feature?

**Comment by KazuCocoa** [comment]
Perhaps 1.18.0.
You can use this via `appium@beta` for now

**Comment by mivola** [comment]
Thx @KazuCocoa 
Unfortunately we're not allowed to use beta versions, so do you think it will make it into an official release soon?

**Comment by KazuCocoa** [comment]
Perhaps no

**Pull request merged**